evap/staff/views.py

```diff
@@ -1586,12 +1586,12 @@ def evaluation_textanswers_update_publish(request):
 def evaluation_textanswers_update_flag(request):
     answer = get_object_from_dict_pk_entry_or_logged_40x(TextAnswer, request.POST, "answer_id")
     assert_textanswer_review_permissions(answer.contribution.evaluation)
-    updated_is_flagged = request.POST.get("is_flagged", None)
-    if updated_is_flagged not in ["true", "false"]:
+
+    is_flagged_bool_string = request.POST.get("is_flagged", None)
+    if is_flagged_bool_string not in ["true", "false"]:
         raise SuspiciousOperation
```

Comment:
codecov says this is not covered, but I feel like I remember seeing a test for invalid inputs here?

Reply:
yep, was broken